keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -263,17 +505,183 @@ When downgrading from a release with this feature, to a release without maxUnava
 - maxUnavailable greater than 1 with partition and staged pods greater than maxUnavailable
 - maxUnavailable greater than 1 with partition and maxUnavailable greater than replicas
 
-## Test Plan
-For `Alpha`, unit tests and e2e tests will be added to test functionality at both
-with feature flag enabled and disabled. Defaults will be verified so that users
-who donot set this flag are not surprised at all.
+New testcases being added:
+
+- Feature enablement/disablement test
+
+Coverage:
+
+- `pkg/apis/apps/v1`:               `2023-05-26` - `71.7%`
+- `pkg/apis/apps/v1beta2`:          `2023-05-26` - `76.7%`
+- `pkg/apis/apps/validation`:       `2023-05-26` - `92.3%`
+- `pkg/controller/statefulset`:     `2023-05-26` - `85.7%`
+- `pkg/registry/apps/statefulset`:  `2023-05-26` - `63.9%`
+
+##### Integration tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+- `test/integration/statefulset`: we'll add a test to verify that the number of pods brought down each
```

Comment:
should also depend on podManagementPolicy here and the number of pods brought down  can be less or equal

Reply:
✅